### DISCUSSION TITLE
Caps_Lock?

### DISCUSSION THREAD

**Discussion by wisim3000** [google/mozc#718] [Q&A]
I was trying to set Caps_Lock as the switching key between Hiragana and Katakana on Fcitx5, however it seems that Mozc cannot recognize it.
Will Mozc supports Caps_Lock in the future?

**Comment by hiroyuki-komatsu** [comment]
Hi wisim3000,

If it's about configurations of Fcitx5, I'd suggest to discuss it in Fcitx's repository.

Talking about configurations of Mozc, Mozc uses Caps_Lock as a modifier key like Shift and Ctrl.
So Caps_Lock cannot be used as a normal key, IIUC.

**Reply by wisim3000** [reply]
Thank you for your reply. 
A Japanese keyboard has its own key to switch between katakana and hiragana. I always thinking it'd be better to use caps lock or something on a normal keyboard to do so...
